shared/teams/team/subteam-row/container.js

```diff
@@ -0,0 +1,72 @@
+// @flow
+import * as I from 'immutable'
+import * as React from 'react'
+import * as Types from '../../../constants/types/teams'
+import * as Constants from '../../../constants/teams'
+import {TeamRow} from '../../main/team-list'
+import {teamsTab} from '../../../constants/tabs'
+import {connect} from 'react-redux'
+import {navigateAppend, navigateTo} from '../../../actions/route-tree'
+import * as KBFSGen from '../../../actions/kbfs-gen'
+import * as RPCTypes from '../../../constants/types/rpc-gen'
+
+import type {TypedState} from '../../../constants/reducer'
+
+type OwnProps = {
+  teamname: string,
+}
+
+type StateProps = {
+  _newTeams: I.Set<string>,
+  _newTeamRequests: I.List<string>,
+  members: number,
+  yourOperations: RPCTypes.TeamOperation,
+  yourRole: ?Types.TeamRoleType,
+}
+
+const mapStateToProps = (state: TypedState, {teamname}: OwnProps): StateProps => ({
+  _newTeams: state.entities.getIn(['teams', 'newTeams'], I.Set()),
+  _newTeamRequests: state.entities.getIn(['teams', 'newTeamRequests'], I.List()),
+  members: state.entities.getIn(['teams', 'teammembercounts', teamname], 0),
+  yourOperations: Constants.getCanPerform(state, teamname),
+  yourRole: Constants.getRole(state, teamname),
+})
+
+type DispatchProps = {
```

Comment:
this is fine but i think not required anymore. flow seems to understand connect() better now and can infer this (usually, use flow coverage etc to make sure)

Reply:
done